example/default_metrics.lua

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env tarantool
+package.path = package.path .. ";../?.lua"
+
+local log = require('log')
+
+-- Create a Metrics Client
+local metrics = require('metrics')
+
+-- Enable default metrics collections
+
+box.cfg{
+    listen = 3302
+}
+
+metrics.default_metrics:collect();
+
+
+-- Init Prometheus Exporter
+local httpd = require('http.server')
+local http_handler = require('metrics.plugins.prometheus').collect_http
```

Comment:
better require module without function resolving

and resolve function in-place

Reply:
that's not my code, and it doesn't has any relations to this pr. I'll create second one with refactor